src/ManageContainer/Client/ManageToDb/Client_test.go

```diff
@@ -176,7 +176,7 @@ func TestGetComputationResultSuccess(t *testing.T) {
 	initialize()
 
 	os.Mkdir(fmt.Sprintf("/Db/result/%s", defaultJobUUID), 0777)
-	data := "{\"id\":\"\",\"job_uuid\":\"m2db_test_jobuuid\",\"result\":\"[\\\"1\\\",\\\"2\\\",\\\"3\\\"]\",\"meta\":{\"piece_id\":1}}"
+	data := "{\"id\":\"\",\"job_uuid\":\"m2db_test_jobuuid\",\"result\":\"[\\\"1\\\",\\\"2\\\",\\\"3\\\"]\",\"meta\":{\"piece_id\":0}}"
```

Comment:
Could you take this opportunity to use raw string literal: `` ` ``?

https://go.dev/ref/spec#String_literals

It is okay to add another refactoring task, and do it at the time.

Reply:
Thank you! You've been very helpful!
I have updated with this PR in 088a14d401a844d9eda7662b259a6528148cd19d as there are few changes.

Reply:
Thank you!  I checked it.